client/src/pages/MyPage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import MyPageMainContainer from '@/containers/MyPageMainContainer';
+import styled from '@emotion/styled';
+import { useRouterContext } from '@/lib/Router';
+import { myPageStateType, PAGE_TYPE } from '@/interfaces/mypage.interface';
+import SideNav from '@/components/MyPage/SideNav';
+import HelloUser from '@/components/MyPage/HelloUser';
+
+const MyPage = (): JSX.Element => {
+  const { params } = useRouterContext();
+  let type = params.type as myPageStateType;
+  if (!(type in PAGE_TYPE)) type = 'main';
```

Comment:
```suggestion
  const type = (type in PAGE_TYPE) ? params.type as myPageStateType : main;
```
음... 좀 난잡한가...
은성님의 선택에 따르겠슴다! 😄

Reply:
`params.type as myPageStateType`를 한 값을 `PAGE_TYPE`으로 체크해야 해서 더 복잡해질 것 같네요😢